contrib/cirrus/setup.sh

```diff
@@ -23,13 +23,14 @@ show_env_vars
 
 req_env_vars AARDVARK_DNS_URL
 
-set -x  # show what's happening
-curl --fail --location -o /tmp/aardvark-dns.zip "$AARDVARK_DNS_URL"
+showrun curl --fail --location -o /tmp/aardvark-dns.zip "$AARDVARK_DNS_URL"
 mkdir -p /usr/libexec/podman
 cd /usr/libexec/podman
 rm -f aardvark-dns*
-unzip -o /tmp/aardvark-dns.zip
+showrun unzip -o /tmp/aardvark-dns.zip
 if [[ $(uname -m) != "x86_64" ]]; then
-    mv aardvark-dns.$(uname -m)-unknown-linux-gnu aardvark-dns
+    showrun mv aardvark-dns.$(uname -m)-unknown-linux-gnu aardvark-dns
 fi
-chmod a+x /usr/libexec/podman/aardvark-dns
+showrun chmod a+x /usr/libexec/podman/aardvark-dns
+
+# Call complete_setup() on EXIT to finalize CI env. vars.
```

Comment:
I don't understand this comment?

Reply:
It's an alert to maintainers that the script isn't "done" at the end, there's an exit handler installed to finalize the setup.  Hmmm, the wording isn't very good is it :cry:   Let me try re-writing it.

Reply:
I should probably add this comment into my aardvark and dhcp PRs too.